src/pages/my-ads/components/AdTypeSection/AdTypeSection.scss

```
@@ -1,7 +1,19 @@
 .ad-type-section {
+    &--edit {
+        padding-top: 2.4rem;
+    }
+    @include desktop {
+        & .radio-group {
+            margin-top: 0;
+        }
+    }
     @include mobile {
         padding: 0 1.6rem;
         overflow: auto;
         max-height: calc(100vh - 27rem);
+
+        &--edit {
+            padding-top: 2.4rem;
+        }
```

is this being affected by the mobile padding: 0 1.6rem? if it is could we do this?
```suggestion
        padding: 2.4rem 1.6rem 0;
        overflow: auto;
        max-height: calc(100vh - 27rem);
```

---

but its needed only for edit.

---

i think i need to change the order of the styles.. i'll update it